lib/logstash/outputs/syslog.rb

```diff
@@ -237,6 +245,14 @@ def setup_ssl
       else
         cert_store.add_file(@ssl_cacert)
       end
+      if @ssl_crl
+        # copy the behavior of X509_load_crl_file() which supports loading bundles of CRLs.
+        File.read(@ssl_crl).split(CRL_END_TAG).each do |crl|
+          crl << CRL_END_TAG
+          cert_store.add_crl(OpenSSL::X509::CRL.new(crl))
```

Comment:
What happens if the CRL file is malformed or contains malformed certificates?

Reply:
It seems to behave similarly as for other malformed .pem files: register will fail.  In this case, register will fail with `OpenSSL::X509::CRLError` with error description "malformed PEM data".